docs/organizer-guide/confs/website.rst

```diff
@@ -0,0 +1,293 @@
+  templates used in rendering the conference pages. This includes the
+  website design, navigation and snippets used in schedule rendering.
+
+The YAML files are validated using the schemas in ``_data/schema-*``.
+
+
+Typical conference workflow
+---------------------------
+
+* The initial conference site is created, with ``flaglanding`` typically
+  set to publish a simple landing page with the conference dates.
+* At some point, the content for the conference is filled in and verified,
+  like sponsorship prices. ``flaglanding`` is now switched off and remains off.
+* As soon as tickets go on sale, ``flagticketsonsale`` is switched on.
+* When the CFP opens, ``flagcfp`` is switched on. Once the CFP has closed,
+  it is switched off and remains off. The CFP details should be in the
+  general conference config file, under ``cfp``.
+* When speakers are announced, ``flagspeakersannounced`` is switched on
+  and remains on. The sessions should be in the
+  ``_data/<shortcode>-<year>-sessions.yaml`` file.
+* When the schedule is ready, ``flaghasschedule`` is switched on and remains on.
+  The schedule should be in the ``_data/<shortcode>-<year>-schedule.yaml`` file.
+* At some point the conference sells out. ``flagticketsonsale`` is switched off,
+  ``flagsoldout`` is switched on.
+* During the conference, if live streaming is available, ``flaglivestreaming``
+  is switched on. It is switched off after the conference ends.
+* After the conference ends, ``flagpostconf`` is switched on and remains on.
+* When videos are published on the Write the Docs website, ``flagvideos``
+  is switched on and remains on.
+
+Each time the state is updated, check the top/bottom buttons to ensure they
+still fit. For example, once tickets have sold out, the button "buy a ticket"
+should be removed.
+
+
+General conference config file
+------------------------------
+
+If you're setting up a new conference, the easiest may be to take a YAML
+config file from the previous conference, and adjust it.
+
+The items in the general conference config file are:
+
+* ``name``: the dynamic part of the name of the conference, e.g. ``Portland``
+  for Write the Docs Portland.
+* ``shortcode``: the shortcode as used in file paths, config names, and other
+  references to the conference, together with the year. Currently used are
+  ``portland``, ``prague``, and ``australia``.
+* ``year``: the conference year, e.g. ``2020``.
+* ``city``: the city for the conference.
+* ``local_area``: the state or country for the conference.
+* ``area``: the wider area for the conference. Currently used are
+  ``North America``, ``Europe``, and ``Australia``.
+* ``area_adj``: the adjective form of the area, e.g. ``Australian``.
+* ``tz``: the timezone the conference is located in.
+* ``email``: general contact email for the conference.
+* ``color``: the accent color for conference branding, e.g. ``blue`` for 2020.
+* ``photos.default``: the default header image.
```

Comment:
How do we choose the picture? Is that OOS for this role?

Reply:
I'd say it's out of scope. We can add that later if we have a process for it.